appium-cookies/index.js

```diff
@@ -0,0 +1,7 @@
+// transpile:main
+
+import {default as cookies} from './lib/cookies.js';
```

Comment:
to distinguish between import/export braces and regular javascript object construction, for import/exports we add a space, as follows:

```js
import { default as cookies } from './lib/cookies.js';
```